parm/config/gfs/config.resources

```diff
@@ -1042,11 +1043,11 @@ case ${step} in
     ;;
 
   "genesis")
-    walltime="00:25:00"
+    walltime="00:40:00"
```

Comment:
Just a note here that if these jobs are going to run longer than 15 mins, then we will need to add restart capabilities.  So while we are experimenting at the full resolution, we should be mindful of what additional work will become open for operational handoff

Reply:
Great point. Thanks!